plugins/source/aws/go.mod

```diff
@@ -123,6 +123,8 @@ require (
 
 require github.com/aws/aws-sdk-go-v2/service/autoscalingplans v1.13.5
 
+require github.com/aws/aws-sdk-go-v2/service/dynamodbstreams v1.14.7
```

Comment:
it's most probably a nit, but could you refactor this? (direct deps into one require, indirect into the second one + `go mod tidy`?